.github/workflows/ci.yml

```diff
@@ -84,6 +84,7 @@ jobs:
       - id: install-sqliteodbc-driver
         run: |
           Invoke-WebRequest -Uri "http://www.ch-werner.de/sqliteodbc/sqliteodbc_w64.exe" -OutFile "sqliteodbc_w64.exe"
+          if ((Get-FileHash 'sqliteodbc_w64.exe').Hash -ne "bd93e8450bcbd4ae16894e06ecb1b8572d6a01448df5b8dd2964eb6d3d302db7") {Fail "Hash mismatch"}
```

Comment:
This works, but it's a bit unorthodox to use the non-existent keyword "Fail" to generate a syntax error in order to stop the CI job.  Would `Write "Hash mismatch"; Exit 1` work?

Reply:
yes, this seems to work, changed it